src/components/SearchPanel/SearchPanel.module.scss

```diff
@@ -0,0 +1,28 @@
+@use '@/assets/variables';
+
+.form {
+  width: 16.8rem;
+  display: flex;
+  justify-content: center;
+  gap: 0.35rem;
+
+  @include variables.media-tablet {
```

Comment:
After applying this
`@use '@/assets/variables' as *;`
You can use variables like that `media-tablet` without `variables.`